api/views/user_views.py

```diff
@@ -213,6 +220,41 @@ def post(self, request):
         return JsonResponse(response, status=status.HTTP_200_OK)
 
 
+@extend_schema(tags=["Authentication Endpoint"])
+class JWTLoginView(APIView):
```

Comment:
Do you think we'll still be using `jwt-token/refresh/` endpoint now that you've added this? 

Just wondering what you think.

Reply:
Yes, I'd imagine there would be a scenario where you'd want an access token reissued so you'd use that instead of logging in from scratch. The more important point is that when you refresh the token, the old token is blacklisted, but when you login any other outstanding refresh tokens are still valid (the hassle-free multiple instance support I was talking about before).